GooglePlayInstant/Samples/SphereBlast/scripts/BaseGame.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using GooglePlayInstant;
+
+public class BaseGame : MonoBehaviour
+{
+	public Transform crate;
+    public Camera thirdPersonCamera;
+    public Camera overheadCamera;
+	private static Vector3 position1 = new Vector3(37, 3.87f, 11);
```

Comment:
Instead of creating these as fields, can you just create them directly in the positions field? E.g.
```
	private static Vector3[] _positions = new Vector3 []
	{
		new Vector3(37.0f, 3.87f, 11.0f),
                new Vector3(37.0f, 3.87f, -4.0f),
...
```

Reply:
yes. That seems reasonable.

Reply:
fixed